stacker/tests/factories.py

```diff
@@ -32,10 +32,13 @@ def mock_context(namespace="default", extra_config_args=None, **kwargs):
     if extra_config_args:
         config_args.update(extra_config_args)
     config = Config(config_args)
-    environment = kwargs.get("environment", {})
+    if kwargs.get("environment"):
```

Comment:
Curious what htis change is about?  Doesn't seem to change much functionally?

Reply:
Oh yeah, this was interesting.

The problem with the current code is that you can't include `environment` in the kwargs, because it's being explicitly provided already. Without the change, it errors like:
```
Traceback (most recent call last):
  File "/home/troyready/python/stacker/stacker/tests/blueprints/test_raw.py", line 156, in test_j2_to_json
    environment={'foo': 'bar'}),
  File "/home/troyready/python/stacker/stacker/tests/factories.py", line 39, in mock_context
    **kwargs)
TypeError: type object got multiple values for keyword argument 'environment'
```

Reply:
Oooh, interesting.  Thanks for the explanation.